src/ray/raylet/node_manager.cc

```diff
@@ -1448,12 +1508,7 @@ void NodeManager::HandleTaskReconstruction(const TaskID &task_id) {
 }
 
 void NodeManager::ResubmitTask(const Task &task) {
-  // Actor reconstruction is turned off by default right now. If this is an
-  // actor task, treat the task as failed and do not resubmit it.
-  if (task.GetTaskSpecification().IsActorTask()) {
-    TreatTaskAsFailed(task.GetTaskSpecification());
-    return;
-  }
+  RAY_LOG(DEBUG) << "Resubmitting task " << task.GetTaskSpecification().TaskId();
```

Comment:
Can we add a check here to make sure that if the task is an actor creation task (`task.GetTaskSpecification().IsActorCreationTask()`), that the actor's current state is `BEING_RECONSTRUCTED`? Just to make sure that reconstruction suppression is working properly.